core/core/src/ansible/roles/filebeat/tasks/main.yml

```diff
@@ -15,3 +15,7 @@
   systemd:
     name: filebeat
     state: started
+
+- name: Verify Filebeat is running
+  command: systemctl is-active filebeat
+  changed_when: false
```

Comment:
This task will never be changed nor failed. Shouldn't we register output/exit code of systemctl and make a condition that'll fail the task if Filebeat is not active?

Reply:
It fails whenever the command returns non-zero exit code (when service is not running), so the verification is correct.